meta/meta.go

```diff
@@ -17,15 +17,15 @@ import (
 //
 // Deprecated: Use Go standard library [runtime/debug] package build information
 // instead.
-var SDKVersion = "2.36.1"
+var SDKVersion = "2.37.0"
```

Comment:
I believe our CI should do this for us? Not sure about the pre-release portion: https://github.com/hashicorp/terraform-plugin-sdk/commit/9a1ad1dbc68125df2491ac25ac1447d21fdf70d2

Reply:
Oh, neat, I did not know!

Reply:
Yeah it looks like it probably won't update pre-release 😆 , not sure exactly who is using this and where, might not be a big deal ATM

https://github.com/hashicorp/terraform-plugin-sdk/blob/9eb7da0be3e9a35771af84fae0a37a53d96fb833/.github/workflows/release.yml#L59-L81

Reply:
I'll update the contributing docs then once we know what'll happen with the pre-release

Reply:
not entirely sure what'll end up in `changelog-version.outputs.version` but I highly suspect the pre-release identifier might be part of it 🤨

Reply:
Oh damn, Github didn't show your comment 😂 So it now seems like I ignored what you said..

Reply:
Here's one data point -- a search across official provider repos. Two providers use this in a user-agent string, which seems fine.
```
$ ag meta\.SDKVersion $(ag -l SDKVersion | grep -v vendor)
terraform-provider-azuread/internal/common/client_options.go
109:	tfUserAgent := fmt.Sprintf("HashiCorp Terraform/%s (+https://www.terraform.io) Terraform Plugin SDK/%s", o.TerraformVersion, meta.SDKVersionString()) //nolint:staticcheck

terraform-provider-azurestack/internal/common/client_options.go
62:	tfUserAgent := fmt.Sprintf("HashiCorp Terraform/%s (+https://www.terraform.io) Terraform Plugin SDK/%s", tfVersion, meta.SDKVersionString())
```

Reply:
Added a commit that fixes the action nevertheless – who knows what else is happening out there 😅

Reply:
I like it! I had not seen the existing `var SDKPrerelease`.